package.json

```diff
@@ -53,7 +53,7 @@
         "@angular/platform-browser-dynamic": "^17.0.7",
         "@angular/router": "^17.0.7",
         "@egjs/hammerjs": "^2.0.17",
-        "@geoengine/openapi-client": "0.0.6",
+        "@geoengine/openapi-client": "https://gitpkg.now.sh/geo-engine/openapi-client/typescript?dataset-info",
```

Comment:
i cannot try it out

> sha512-C0qeNnkj64nZer0RpCAQTkJ9Ep6YLfjGkYeFEuJjIyc4+q7dYkHY+rpwcbw72lLWJh0VN4JDn8SEwhrlpMYuiQ== integrity checksum failed when using sha512: wanted sha512-C0qeNnkj64nZer0RpCAQTkJ9Ep6YLfjGkYeFEuJjIyc4+q7dYkHY+rpwcbw72lLWJh0VN4JDn8SEwhrlpMYuiQ== but got sha512-GYoRV7sWNW+URz70266eA+lZdkJSgqXArzMCxC5kjkD7ljgaIsO1SFqBZyv0i4x0nQv+Flx5hLrnMKSZ/pcS0g==. (240921 bytes)

Reply:
strange that it works only on my machine. I have to idea how to fix this.

Maybe we could instead release a geoengine-ui in-development package on npm instead?

Reply:
I can now compile the common module, but still not the manager:

> ⠏ Building...
✘ [ERROR] Could not resolve "dist/core/assets/fonts/poppins/poppins.css"
>    angular:styles/global:styles:2:8:
      2 │ @import 'dist/core/assets/fonts/poppins/poppins.css';

The problem is, that you defined the manager like this:

```json
…
                        "styles": [
                            "projects/manager/src/styles.scss",
                            "dist/core/assets/fonts/poppins/poppins.css",
                            "dist/core/assets/fonts/pacifico/pacifico.css",
                            "dist/core/assets/fonts/material-design-icons/material-icons.css"
                        ],
…
```

But this only works if you builder core before. But common is a dependency of core and not the other way around.

Reply:
52af159d83445fa69a5dd5be9aa1df587cb0de07